src/Modals/Modal.tsx

```diff
@@ -115,7 +115,9 @@ const StyledModal = styled.div<{theme: BaseTheme}>`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  max-width: 80%;
+  min-width: 40em;
+  max-width: 40em;
```

Comment:
I discussed this with Vittorio, and he advised to remove the `max-width` for now (commit 64cfae1) so that the modal can stretch and noted to discuss media queries in our next sprint meeting. https://github.com/seas-computing/course-planner_ux/commit/3478b6bd584329de3eb7c6932256e6e8a49c1f3d#diff-f911b13fed62245efc0a3b65188aa861